src/scripts/main.js

```diff
@@ -1,13 +1,30 @@
 'use strict';
 
+const form = document.querySelector('form');
+
 function waitFor(element, eventName) {
```

Comment:
Function `waitFor` is a utility function and its name could be more descriptive. Consider renaming it to something like `waitForEventOnElement` to make it more clear what it does.